common/types/envconfig.go

```diff
@@ -22,4 +31,5 @@ type EnvConfig struct {
 	SidecarSyncProvider  string `envconfig:"SIDECAR_SYNC_PROVIDER" default:"kubernetes"`
 	SidecarLogFormat     string `envconfig:"SIDECAR_LOG_FORMAT" default:"json"`
 	SidecarProbesEnabled bool   `envconfig:"SIDECAR_PROBES_ENABLED" default:"true"`
+	FlagdResourceEnabled bool   `envconfig:"FLAGD_RESOURCE_ENABLED" default:"true"`
```

Comment:
nit - we should document this env flag and guide on how to use it

Reply:
this is documented in the related helm value here: https://github.com/open-feature/open-feature-operator/pull/633/files#diff-c1955654bbc791643783433fa9cc013ea352647c08712007bb5af430ca398b7bR113 

The reason why this flag has been added is because handling this new CRD requires additional permissions for the operator, which may not be something users would like to grant if they do not intend to make use of this new feature

Reply:
Okay, yes can disable this for permission reasons. That's a valid point 👍